src/server/common/helpers/auth/get-bell-options.js

```diff
@@ -0,0 +1,39 @@
+import Jwt from '@hapi/jwt'
+import { config } from '#config/config.js'
+
+export function getBellOptions(oidcConfig) {
+  return {
+    provider: {
+      name: 'entra-id',
+      protocol: 'oauth2',
+      useParamsAuth: true,
+      auth: oidcConfig.authorization_endpoint,
+      token: oidcConfig.token_endpoint,
+      scope: ['openid', 'profile', 'email'],
+      profile: function (credentials, _params, _get) {
+        const payload = Jwt.token.decode(credentials.token).decoded.payload
+
+        credentials.profile = {
+          ...payload,
+          displayName:
+            payload.name ||
+            `${payload.given_name || ''} ${payload.family_name || ''}`.trim()
```

Comment:
Messier, but ensures there aren't multiple whitespace chars between first and last name
```suggestion
            `${(payload.given_name || '').trim()} ${(payload.family_name || '').trim()}`
```

although I think there's `(payload.name || '').trim()` where the `name` property already concats the `given_name` and `family_name`